package.json

```diff
@@ -71,14 +71,12 @@
     "redux-mock-store": "1.5.4",
     "redux-thunk": "2.3.0",
     "regenerator-runtime": "0.13.7",
-    "sanitize-html": "2.8.1",
-    "scheduler": "^0.23.0",
-    "timeago.js": "^4.0.2",
+    "sanitize-html": "2.12.1",
+    "timeago.js": "4.0.2",
     "universal-cookie": "4.0.4",
-    "url": "0.11.0",
     "use-context-selector": "^1.4.1",
     "uuid": "9.0.0",
-    "validator": "10.11.0"
+    "validator": "13.7.0"
```

Comment:
[sanity check] Are there any breaking changes that this MFE needs to address as part of the upgrade from v10 to v13?

Reply:
Based on the release notes and commits, there doesn't appear to be any breaking changes. Just mainly enhanced localizations and new features.
https://github.com/validatorjs/validator.js/compare/10.11.0...13.7.0
https://github.com/validatorjs/validator.js/releases?page=1
But I'll take some extra time to test those code paths that specifically use this package 👍🏽